src/lib/bootloader/exceptions.rb

```diff
@@ -26,6 +26,23 @@ def initialize(msg)
     end
   end
 
+  # Represent unsupported value in given option. Used mainly when value contain something that
+  # bootloader does not understand yet.
+  class UnsupportedOption < RuntimeError
+    include Yast::I18n
+    attr_reader :option
+
+    def initialize(option)
+      @reason = option
+      textdomain "bootloader"
+
+      # TRANSLATORS: %s is name of option that has unsupported value
+      super _("Option '%s' contain unsupported value.\nYaST2 Bootloader cannot handle it.\n" \
+        "To use YaST2 Bootloader configuration have to be reproposed\n" \
```

Comment:
IMHO this part is too complicated, simpler

`The configuration needs to be proposed again or edited manually.` 

would be better.